### PR TITLE
#DearInternet page polish

### LINKED ISSUE #5659: Add polish / effects to #DearInternet Page
Because of the tight deadline, we should aim for getting the MVP version of #DearInternet Page built (#5616 ). If time allows, add polish and effects to the page to make it shine more.

More details to come once we have finalized design.

**Things we can polish ideas:**
- [x] Plane path: streamline start and end _(extend path?)_
   - update svg with a longer path (https://drive.google.com/open?id=1FJ5oEcUrMwVH68YStnpuRhv5wjiTRROo&authuser=sabrina%40mozillafoundation.org&usp=drive_fs)
- [x] Plane: scroll animation
  - similiar to what mavis has in the codepen 
- [x] adapt share buttons on mobile _(currently none)_
   - put them before the 3 envelopes in the CTA section
- [x] adjust heading/love letter graphic size on mobile _(to be a bit larger)_
  - enlarge them to full width on small mobile
- [x] h1 sizing on mobile
  - enlarge it a bit
- [x] adjust heart path top margin on larger desktop _(still a bit close to last letter)_
- [x] any accessibility improvements we can make?
  - 5th cloud (light-pink): change it to light-blue
  - very last cloud (light-blue): change it to white

### PR DESCRIPTION
**Pull request by mmmavis**
(will assign code reviewer once this passes design review)

---

Closes #5659 
Add #DearInternet page polish + remove unused SVG files.

https://foundation-s-issue-5659-igica4.herokuapp.com/en/campaigns/dearinternet/

Note for code reviewer: there are a bunch white space differences in the PR.

### CODE REVIEW

**Review by sabrinang — CHANGES_REQUESTED** [review]
770px-990px screens the share buttons and envelopes are missing (should be mobile style then)

The animation is looking great!

**Comment by mmmavis** [comment]
> 770px-990px screens the share buttons and envelopes are missing (should be mobile style then)

Good catch! Since the rest of the page changes layout at 768px breakpoint, I'm gonna use desktop style for 768px+ screen so we don't have to change existing implementaion much. If things look too awful we can chat and see what minimal changes we can do to make them better.

**Review by sabrinang — APPROVED** [review]
Great thanks for incorporating the social share buttons back for those breakpoints!

File: source/js/foundation/pages/dear-internet/template-js-handler/plane.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        this.defaultPlaneRotation = PLANE.getAttribute("transform")
+          .match(regex)[0]
+          .replace("rotate", "")
+          .replace("(", "")
+          .replace(")", "")
+          .split(" ")[0]
+          .trim();
+      }
+      this.defaultPlaneRotation = parseFloat(this.defaultPlaneRotation);
+    }
```
Comment (Pomax) [inline source/js/foundation/pages/dear-internet/template-js-handler/plane.js:75]: We can use a regex capture group here:

```suggestion
    if (PLANE.getAttribute("transform")) {
      // Get the first rotation value out of the transforms.
      // Example of "transform" attribute value:
      // "translate(num num num) rotate(num num num)"
      const match = PLANE.getAttribute("transform").match(/rotate\((\S+)[ )]/)[1];
      if (match) {
        this.defaultPlaneRotation = parseFloat(match[1]);
      }
    }
```

If we only care about the first rotation argument, we can actually look for "not whitespace" after the opening parenthesis, and then if that exists, parse it as number.

```
rotate   - just the word
\(       - literal (
(\S+)    - one or more "not whitespace" characters, captured as a group
[ )]     - either a space, or literal )
```

File: source/js/foundation/pages/dear-internet/template-js-handler/plane.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    if (!SVG_WRAPPER || !SVG || !PATH || !PLANE) {
+      console.error(`Some DOM elements are missing`);
+      return;
+    }
+
+    this.findDefaultPlaneRotation();
+
+    window.addEventListener("scroll", () => {
+      this.movePlane();
+    });
```
Comment (Pomax) [inline source/js/foundation/pages/dear-internet/template-js-handler/plane.js:88]: Remember to always mark scroll event handlers as "passive" so they don't interfere with the actual scrolling. 
```suggestion
    window.addEventListener("scroll", () => this.movePlane(), { passive: true });
```

**Review by Pomax — CHANGES_REQUESTED** [review]

File: source/js/foundation/pages/dear-internet/template-js-handler/plane.js (hunk 1 above)
Comment (mmmavis) [reply source/js/foundation/pages/dear-internet/template-js-handler/plane.js:75]: Ahhhh you are right! Thanks for the explanation too. Totally forgot that we can grab the first argument directly from regex too lol 🙈

File: network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/dear_internet_plane_path.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        <path id="to-use-o" d="M39.7005,1.69875 L33.18975,20.1435 C32.994,20.697 32.946,21.2925 33.051,21.87 L33.051,21.87 L36.01125,38.14875 C36.10875,38.69625 35.97375,39.23625 35.66625,39.64875 L35.66625,39.64875 C35.3655,40.068 34.89375,40.36125 34.34625,40.43625 L34.34625,40.43625 L17.955,42.5775 C17.37225,42.65325 16.8195,42.8805 16.3515,43.23525 L16.3515,43.23525 L0.77625,55.053 C-0.55875,56.0655 -0.08625,58.17375 1.548,58.51875 L1.548,58.51875 L81.8445,75.591 C83.493,75.9405 84.78075,74.178 83.94525,72.71475 L83.94525,72.71475 L43.23375,1.383 C42.852,0.717 42.195,0.39675 41.54175,0.39675 L41.54175,0.39675 C40.76925,0.39675 40.00125,0.84525 39.7005,1.69875 L39.7005,1.69875 Z"/>
```
[the remark this answers is not in the document]
Comment (mmmavis) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/dear_internet_plane_path.html:10]: These were generated by Sketch and I just did some clean-up to remove extra metadata.

I'll tidy up this file further based on your suggestion!

**Review by Pomax — APPROVED** [review]
this is so nice!

**Comment by mmmavis** [comment]
Yay!

**Pull request merged**